integration/test-data-source/main.tf

```diff
@@ -0,0 +1,65 @@
+terraform {
```

Comment:
nit: curious if we can use a file from the examples directory, or put it there.

Reply:
We could use some, but most have dependencies like gcloud credentials that we can't necessarily provide.